src/components/CommandBar/CommandBarHeaderFooter.tsx

```diff
@@ -38,6 +42,10 @@ function CommandBarHeaderFooter({
   }, [selectedCommand])
   const isReviewing = commandBarState.matches('Review')
   const [showShortcuts, setShowShortcuts] = useState(false)
+  const isCurrentArgRequired =
```

Comment:
Nice, I support this, this was the only thing I was curious about. Good thinking

Reply:
Dope! Thank you. Added the pw test steps, this is ready to go now